apache-mesos-mixin/README.md

```diff
@@ -60,8 +60,8 @@ scrape_configs:
           __path__: /var/log/mesos/agent/*.log*
 ```
 
-![Apache Mesos Overview Dashboard 1](TODO)
-![Apache Mesos Overview Dashboard 2](TODO)
+![First screenshot of the Apache Mesos overview 1](https://storage.googleapis.com/grafanalabs-integration-assets/apache-mesos/screenshots/apache_mesos_overview_1.png)
+![Second screenshot of the Apache Mesos overview 2](https://storage.googleapis.com/grafanalabs-integration-assets/apache-mesos/screenshots/apache_mesos_overview_2.png)
```

Comment:
Minor nitpick:

"first screenshot" and "1" in the same title is very redundant, I'd drop either the description or the number, to only keep one.
Same for the second screenshot